AccountingSystem/Service/DataStoreHandler.py

```diff
@@ -21,11 +22,14 @@ class DataStoreHandler( RequestHandler ):
   def initializeHandler( cls, svcInfoDict ):
     multiPath = PathFinder.getDatabaseSection( "Accounting/MultiDB" )
     cls.__acDB = MultiAccountingDB( multiPath )
-    cls.__acDB.autoCompactDB() #pylint: disable=no-member
-    result = cls.__acDB.markAllPendingRecordsAsNotTaken() #pylint: disable=no-member
-    if not result[ 'OK' ]:
-      return result
-    gThreadScheduler.addPeriodicTask( 60, cls.__acDB.loadPendingRecords ) #pylint: disable=no-member
+    #we can run multiple services in read only mode. In that case we do not bucket
+    cls.runBucketing = getServiceOption( svcInfoDict, 'RunBucketing', True )
+    if cls.runBucketing:
+      cls.__acDB.autoCompactDB() #pylint: disable=no-member
+      result = cls.__acDB.markAllPendingRecordsAsNotTaken() #pylint: disable=no-member
```

Comment:
yes, autoCompactDB does the work! but we do not want to change the status of pending requests also, if one of the service restart....